test/ant-test-files/non-ant-project/src/several/namespace/layers/A.java

```diff
@@ -0,0 +1,2 @@
+package several.namespace.layers;
+public class A{}
```

Comment:
You can delete all of the `.java` files and the files that need to exist for the purposes of tests can be 0 bytes in size. The contents of the files don't matter.

Reply:
Deleted the Java files in a02926ff87cd9b8a94ce70e005b52e34100bc9a3